File: readJSON.py
import json

with open('books.json') as file:
    data = json.load(file)

first_book_title = data['Books'][0]['title']
print("Title of the first book:", first_book_title)

second_book_author = data['Books'][1]['author']
print("Author of the second book:", second_book_author)

all_genres = [genre for book in data['Books'] for genre in book['genres']]
print("Genres of all books:", ", ".join(all_genres))

File: readJSON.js
const fs = require('fs');

fs.readFile('books.json', 'utf8', (err, data) => {
  if (err) {
    console.error("Error reading file:", err);
    return;
  }

  const bookData = JSON.parse(data);

  const firstBookTitle = bookData.Books[0].title;
  const secondBookAuthor = bookData.Books[1].author;
  const allGenres = bookData.Books.map(book => book.genres).flat();

  console.log("Title of the first book:", firstBookTitle);
  console.log("Author of the second book:", secondBookAuthor);
  console.log("Genres of all books:", allGenres.join(", "));
});
